docs/useful-robot-commands.md

```diff
@@ -42,6 +42,14 @@ This will stop the current Systemd services, replace and restart them. Binaries
 
 `bazel run //software/jetson_nano/ansible:run_ansible --cpu=jetson_nano -- --playbook deploy_nano.yml --hosts <robot_ip> --ssh_pass <jetson_nano_password>`
 
+You could also use the `tbots.py` script to flash
+
+`./tbots.py run run_ansible -f <robot_ids> -pwd <jetson_nano_password>` (Note that this uses robot IDs rather than full robot IP addresses)
+
+Example: Flashing robots 1, 4, and 7
+
+`./tbots.py run run_ansible -f 1 4 7 -pwd <jetson_nano_password>`
+
 ## Flashing the powerboard
 
 This will flash powerloop, the current firmware in `software/power/`, onto the power board. It will prompt the user into setting the powerboard into bootloader mode by holding the reset button and pressing the boot button. Then once the board is flashed, pressing the reset button after to use the new firmware.  
```

Comment:
Is this going to flash powerloop every time? 

Also I think you have the buttons backwards according to this message from Dannon in #robot_integration
![Screenshot_20230624-193722.png](https://github.com/UBC-Thunderbots/Software/assets/48522062/8b03ba1f-8a49-4fcb-a0ea-470070c0b81c)

Reply:
So flashing powerloop is done separately and not done through the `deploy_nano` script. 
But regarding the button presses, I'm personally not sure. @LiCody Can you confirm if the docs is correct?

Reply:
Ah whoops rookie mistake - that's not even part of the PR.

Reply:
oh the docs are wrong should be hold boot press reset

Reply:
I've updated the docs

Reply:
Thanks for updating even tho it wasn't ur change!